examples/vortex-mpi.py

```diff
@@ -52,20 +53,45 @@
 from mirgecom.initializers import Vortex2D
 from mirgecom.eos import IdealSingleGas
 
+from logpyle import (LogManager, IntervalTimer, add_general_quantities,
+        add_simulation_quantities, add_run_info)
+
+from mirgecom.logging_quantities import (DependentDiscretizationBasedQuantity,
+    ConservedDiscretizationBasedQuantity, KernelProfile, add_package_versions)
+
 
 logger = logging.getLogger(__name__)
 
 
 @mpi_entry_point
-def main(ctx_factory=cl.create_some_context):
+def main(ctx_factory=cl.create_some_context, use_profiling=False, use_logmgr=False):
     """Drive the example."""
+    from mpi4py import MPI
+    comm = MPI.COMM_WORLD
+
+    if use_logmgr:
+        logmgr = LogManager("vortex.dat", "wu", mpi_comm=comm)
+        add_run_info(logmgr)
+        add_package_versions(logmgr)
+        add_general_quantities(logmgr)
+        add_simulation_quantities(logmgr)
```

Comment:
I was thinking of adding this into a new class `MirgecomLogManager` or so. What do you think?

Also, why are some functions methods of the class, but some are global functions, e.g. 

`add_run_info(logmgr)`, but `logmgr.add_quantity(...)`